pkg/visor/visorconfig/values.go

```diff
@@ -180,12 +180,23 @@ func DmsgPtyWhiteList() string {
 func MustPK(pk string) cipher.PubKey {
 	var sPK cipher.PubKey
 	if err := sPK.UnmarshalText([]byte(pk)); err != nil {
+		fmt.Printf("invalid public key: %s", pk)
 		panic(err)
 	}
 
 	return sPK
 }
 
+// MustPKs unmarshals comma separated list of string PKs to []cipher.PubKey. It panics if unmarshaling fails.
+func MustPKs(pks string) []cipher.PubKey {
+	var sPKs cipher.PubKeys
+	if err := sPKs.Set(pks); err != nil {
+		fmt.Printf("invalid public key or keys: %s", pks)
```

Comment:
I see that this package uses `fmt` consistently. Is there a reason we are using `fmt` here instead of the logging pacakge we use typically?

Reply:
There is no logging package already used in this particular file, but fmt was already imported

i was replacing a panic with an actual error before the panic because at one point i was hitting that panic and it was very unhelpful.

Reply:
@0pcom we use the logging package liberally throughout the project. I dont see a reason we should avoid importing it here unless you see an explicit reason.